tests/README.md

```diff
@@ -182,27 +148,45 @@ expected output. The framework includes the following built-in verify functions:
    1. For each line in `expected` with the prefix "- " there must be no line in
       the output containing the following string.
 
-1. `_verify_failure`** `func`
+1. **`_verify_failure`** `func`
 
    Runs `func` and confirms that it fails (i.e., non-zero return code). This function is useful
    for testing commands that demonstrate configurations that are expected to fail.
 
-## Running the Tests: Make
+After all test steps are run, add
+```sh
+# @cleanup
+```
+on a single line, followed by the cleanup steps that clean up all the resources. These steps can also directly call functions defined in the auto-sourced scripts described before, as well as any script you have sourced by yourself for the test steps.
 
-You can execute all istio.io tests using make.
+## Running The Tests
 
+Run
 ```bash
-make test.kube.presubmit
+make doc.test
 ```
+to start testing all docs in the content folder within a `kube` environment. This command takes three optional environment variables: `TEST`, `TIMEOUT` and `ENV`.
 
-Alternatively, you can run the tests in a particular package under `tests/`.
-For example, the following command will only run the traffic management tests:
-
+`TEST` specifies the tests to be run using the name of the directory. For example, the command
+```bash
+make doc.test TEST=traffic-management
+```
+will run all the tests under `traffic-management` folder. The `TEST` variable also accepts multiple test names separated by commas, for example,
```

Comment:
run all tests under `traffic-managment` folder makes no sense. What `traffic-managment` folder? There are several. I think you should require a full path to the doc relative to `content/en/docs`. So it could be `TEST=tasks/traffic-managment` or `TEST=ops/configuration/traffic-managment` or `TEST=ops/best-practices/traffic-management` or other. Just giving the short name is confusing and not helpful.

Reply:
How about [this](https://github.com/istio/istio.io/pull/7465/files#diff-62f0ff60ede3b78b9c95624e2f61d6c1R169)? I suggest it would be the recommended practice to specify full paths.

Reply:
If you allow anything other than the full path, you need to explain how it works, and to be honest, I can't see a sensible answer. What happens if the user says "TEST=foo", or "TEST=foo/bar", or "TEST=foo/bar/baz"? Are you going to scan the entire repo looking for all parent directories that have the specified subdir(s)? It seems like a big complication for a small gain. Users will typically be using this feature to test a doc that they know the full path for, so copy and pasting that after the `TEST=` part of the command is pretty easy and straightforward.

Reply:
The current way of matching tests is to traverse through the entire folder and check if a path contains ${TEST} as a substring, so ``foo/bar/baz`` will match short paths like `foo`, `bar`, `foo/bar` and also the full path `foo/bar/baz`. Okay I see this is kind of confusing once I explain it... let me change that.

Reply:
This is now done by the latest commit. Take a look!